xbmc/pictures/SlideShowPicture.h

```diff
@@ -136,4 +136,6 @@ class CSlideShowPic
   Microsoft::WRL::ComPtr<ID3D11Buffer> m_vb;
   bool UpdateVertexBuffer(Vertex *vertices);
 #endif
+
+  uint32_t m_vao;
```

Comment:
This member (and the one added in RPRendererGuiTexture.h) needs to be guarded by `#if defined(HAS_GL)` to avoid an unused variable warning on GLES:

```
SlideShowPicture.h:140:12: warning: private field 'm_vao' is not used [-Wunused-private-field]
  uint32_t m_vao;
           ^
```